flutter_appauth_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# Changelog
```

Comment:
can you please revert this change as it's not necessary and not even a convention being used within this repo. If you look at the various plugins in the community, you'll find this isn't added and in fact redundant on pub.dev when there's a "Changelog" tab with the same text as shown below

<img width="836" alt="Screenshot 2022-12-03 at 9 11 35 pm" src="https://user-images.githubusercontent.com/25263378/205435784-a9b84219-8c09-46d1-8268-eda00eb5e56f.png">

Reply:
Done